sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -387,6 +387,25 @@ describe("LocalWorkspace", () => {
 
         await stack.workspace.removeStack(stackName);
     });
+    it(`refreshes before preview`, async() => {
+        // We create a simple program, and scan the output for an indication
+        // that adding refresh: true will perfrom a refresh operation.
+        const program = async () => {
+            return {
+                toggle: true,
+            };
+        };
+        const projectName = "inline_node";
+        const stackName = fullyQualifiedStackName(getTestOrg(), projectName, `int_test${getTestSuffix()}`);
+        const stack = await LocalWorkspace.createStack({ stackName, projectName, program });
+        const refresh = true;
+        // • First, run Up so we can set the initial state.
+        await stack.up({ userAgent });
+        // • Next, run preview with refresh and check that the refresh was performed.
+        const previewRes = await stack.preview({ userAgent, refresh });
```

Comment:
nit: probably best to define refresh closer to the preview since it is specifically not meant to be used in up.
```suggestion
        // • First, run Up so we can set the initial state.
        await stack.up({ userAgent });
        // • Next, run preview with refresh and check that the refresh was performed.
        const refresh = true;
        const previewRes = await stack.preview({ userAgent, refresh });
```

Reply:
I always dream of refactoring so the fields are alphabetized, but I always forget by the time I submit the PR ;D